debian/eos-default-background.install

```diff
@@ -1 +1,3 @@
 usr/share/eos-media/desktop-background-C.jpg
+usr/share/eos-media/desktop-background-dark-C.jpg
+usr/share/gnome-background-properties/endlessos-dynamic.xml
```

Comment:
Unfortunately this causes an error at package install time because this file is in both packages:

```
dpkg: error processing archive eos-media_1.4.0_all.deb (--install):
 trying to overwrite '/usr/share/gnome-background-properties/endlessos-dynamic.xml', which is also in package eos-default-background 1.4.0
```

I think you can fix this by adjusting `eos-media.install` thusly:

```
-usr/share/gnome-background-properties
+usr/share/gnome-background-properties/endlessos-backgrounds.xml
```

Reply:
I tested this as follows:

```
toolbox enter eos-toolbox-latest
dpkg-buildpackage -uc -us -b
exit
cd ..
sudo eos-dev-unlock
sudo dpkg -i eos-default-background_1.4.0_all.deb eos-media_1.4.0_all.deb
```